<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidgetMain">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="pushButtonModifier_Menu">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter_Menu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSupprimer_Menu">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>33</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Materiel</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Id du badge</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Futur controle</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonLireBadge_Menu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lire le badge</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="labelIdBadge_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Id du badge:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomMateriel_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du materiel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelType_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIdBadge_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNomMateriel_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxType_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelControle_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dernier contr√¥le:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReset_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAjouterType_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;a href=&quot;#&quot;&gt;Besoin de rajouter un type?&lt;/a&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditControle_OperationMateriel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonModifier_OperationMateriel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNewDateControle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNewDateControle">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de jour 
entre 2 contr√¥les:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QPushButton" name="pushButtonValider_OperationType">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAnnuler_OperationType">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomType_OperationType">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNomType_OperationType">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelNomImage_OperationType">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNomImage_OperationType">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOpen_OperationType">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/ressources/images/open.png</normaloff>:/ressources/images/open.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionVoirMateriel"/>
    <addaction name="actionType"/>
    <addaction name="actionAdmin"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVoirMateriel"/>
   <addaction name="separator"/>
   <addaction name="actionType"/>
   <addaction name="separator"/>
   <addaction name="actionAdmin"/>
  </widget>
  <action name="actionVoirMateriel">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/voirInfo.png</normaloff>:/ressources/images/voirInfo.png</iconset>
   </property>
   <property name="text">
    <string>Voir le mat√©riel</string>
   </property>
  </action>
  <action name="actionType">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/type.png</normaloff>:/ressources/images/type.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter un type</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/ressources/images/admin.png</normaloff>:/ressources/images/admin.png</iconset>
   </property>
   <property name="text">
    <string>G√©rer l'administration</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
